CONTRIBUTING.md

```diff
@@ -37,6 +37,8 @@ e.g. to build all workspaces together, run the following command from the root:
 yarn build
 ```
 
+**NOTE:** To avoid build issues, we recommend using [Node](https://nodejs.org/en/download/current/) *v16.x (16+)* and not *v17.x (17+)*. You can install and use [nvm](https://github.com/nvm-sh/nvm#intro) to change your node version.
```

Comment:
This is the wrong place for this note. This section is talking about the monorepo workflow and has nothing to do with node.

I think you can just add "Node 16" in line 15 where it mentions git and yarn.

And there is no need to mention `nvm` (in fact, it is better to use [`fnm`](https://github.com/Schniz/fnm)) as it's cross platform). It's upto the contributor how they want to manage their versions.

Lastly, "16+" implies 17 and 18, so you should just say "16".

Reply:
**[DONE]** Thanks, I just mentioned [Node 16](https://nodejs.org/en/download/current/) on line 15.

I had placed it earlier on line 40 since with v17, I think there were no issues with `yarn install`. But it started causing issues when running `yarn build` (not sure). So I placed it near the mention of `yarn build`